Shared/UserAgent.swift

```diff
@@ -108,9 +109,7 @@ public struct CustomUserAgentConstant {
     public static let customUAFor = [
         "paypal.com": defaultMobileUA,
         "yahoo.com": defaultMobileUA,
-        "disneyplus.com": customDesktopUA,
-        URLProvider.production.domain: ecosiaMobileUA,
```

Comment:
I guess `ecosiaMobileUA` variable is no longer used in this class' context, right? In that case it can be removed 🧹